src/ducks/apps/index.js

```diff
@@ -182,10 +182,7 @@ export function getInstalledApps(state) {
 }
 
 export function getRegistryApps(state) {
-  // display only apps with stable versions for now
-  return state.apps.list
-    .filter(app => app.isInRegistry)
-    .filter(app => Array.isArray(app.versions.stable) && !!app.versions.stable)
+  return state.apps.list.filter(app => app.isInRegistry)
```

Comment:
The second filter was unnecessary since the Store already checked for the presence of versions. This filter was just breaking things if we use a channel different from `stable` finally. The tests about this selector still pass (another proof that this filter was useless)